rank/pages/index.tsx

```diff
@@ -1,30 +1,24 @@
-import { GetServerSideProps, NextPage } from 'next'
-import { SearchTemplate, getTemplates } from '../services/search-templates'
-import { removeEmpty } from '../utils'
-import SearchTemplateTests from '../components/SearchTemplateTests'
+import { GetStaticProps, NextPage } from 'next'
+import { SearchTemplate, SearchTemplateString } from '../types/searchTemplate'
+import { getTemplates, listIndices } from '../services/search-templates'
+
 import { H1 } from '../components/H'
+import Link from 'next/link'
+import SearchTemplateTests from '../components/SearchTemplateTests'
+import { removeEmpty } from '../services/utils'
 
 type Props = {
   searchTemplates: SearchTemplate[]
 }
 
-function get(str: string) {
-  const [env, namespace, index] = str.split('/')
-  return { env, namespace, index }
-}
-
-export const getServerSideProps: GetServerSideProps<Props> = async ({
-  query,
-}) => {
-  const tests = (
-    Array.isArray(query.test) ? query.test : query.test ? [query.test] : []
-  ).map((test) => get(test))
-
-  const searchTemplates = await getTemplates({ prod: true })
+export const getStaticProps: GetStaticProps<Props> = async () => {
```

Comment:
This will mean that we don't update except on build, but we can fix that in a perf PR.